packages/vite/src/node/__tests__/plugins/css.spec.ts

```diff
@@ -110,6 +110,49 @@ position: fixed;
     expect(result1.code).toBe(result2.code)
     resetMock()
   })
+
+  // following tests requires `sass` as devDependencies.But when we add sass to devDependencies,types is conflicts between @types/sass and sass itself types.
```

Comment:
As I mentioned in the comments, following tests requires `sass` as devDependencies.But when we add sass to devDependencies,types is conflicts between @types/sass and sass itself types.

any ideas?

Reply:
ah, it would be good to put the tests in the playground directory.